db/migrate/20250722071916_change_gender_to_string_in_users.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeGenderToStringInUsers < ActiveRecord::Migration[7.0]
+  def change
+    change_column :users, :gender, :integer, using: 'CAST(gender AS INTEGER)'
```

Comment:
```suggestion
    change_column :users, :gender, :integer
```
Suggestion: Chỗ này để tránh ảnh hưởng thì e cũng có thể drop và tạo lại cột